### PR TITLE
Restructure and reuse of USB instructions

### PR DESCRIPTION
**Pull request by rcaballeromx**
Based on Bun's original commit, this pull request addresses all issues.
It also point to the correct branch for merging.

### CODE REVIEW

File: source/clear-linux/get-started/bootable-usb/bootable-usb.rst [hunk 1]
```diff
@@ -3,12 +3,30 @@
...
+
+With the appropriate image downloaded, choose the step-by-step instructions
+applicable to your system:
 
 .. toctree::
     :maxdepth: 1
 
     bootable-usb-linux
     bootable-usb-windows
     bootable-usb-mac
```
Comment (bktan8) [inline source/clear-linux/get-started/bootable-usb/bootable-usb.rst]: This looks fine in RST, but when rendered it looks like this:

With the appropriate image downloaded, choose the step-by-step instructions applicable to your system:

Create a bootable USB drive on Linux
Create a bootable USB drive on Windows
Verify the integrity of the Clear Linux image
Uncompress the Clear Linux image
Burn the Clear Linux image onto a USB drive
Next steps
Create a bootable USB drive on macOS

The "verify", "uncompress", "burn", and "next steps" shouldn't be there.

File: source/clear-linux/get-started/bootable-usb/bootable-usb-mac.rst [hunk 2]
```diff
@@ -1,76 +1,65 @@
...
 
 Create a bootable USB drive on macOS
-************************************
+####################################
 
 Follow these instructions to create a bootable |CLOSIA| USB drive.
 Use an **8GB** or larger USB drive.
 
+These instructions assume you have
+:ref:`downloaded and appropriate image<download-usb-image>`.
```
Comment (bktan8) [inline source/clear-linux/get-started/bootable-usb/bootable-usb-mac.rst]: the "and" should be "an"

File: source/clear-linux/get-started/bootable-usb/bootable-usb.rst (hunk 1 above)
Comment (rcaballeromx) [reply source/clear-linux/get-started/bootable-usb/bootable-usb.rst]: This issue was being caused by the incorrect heading underline on the origninal.

File: source/clear-linux/get-started/bootable-usb/bootable-usb-mac.rst (hunk 2 above)
Comment (rcaballeromx) [reply source/clear-linux/get-started/bootable-usb/bootable-usb-mac.rst]: Fixed all instances

**Comment by rcaballeromx** [comment]
@tpleavitt Bun wishes to add some changes which got lost during clean up. I'll hold the merge until those are included.

**Comment by bktan8** [comment]
@rcaballeromx - I am finished adding back the stuff that was lost from my push before Xmas.

**Comment by bktan8** [comment]
LGTM

**Comment by bktan8** [comment]
My bad.  I missed the period "." between my name and Tom's.  Thought you were asking me to approve.

**Pull request merged**